### PR TITLE
Create syscalls file

### PR DESCRIPTION
**Pull request by curiosity-seeker**
A little script to determine the necessary syscalls for a program.

### CODE REVIEW

File: contrib/syscalls [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```
Comment (rusty-snake) [inline contrib/syscalls]: `contrib/syscalls` -> `contrib/syscalls.sh`

**Comment by Vincent43** [comment]
@curiosity-seeker please double quote all variables.

**Comment by curiosity-seeker** [comment]
> @curiosity-seeker please double quote all variables.

I'm not sure that I understand correctly. To my knowledge awk and sed do not support double quotes.

**Comment by rusty-snake** [comment]
@curiosity-seeker my understanding: 
`cat $SYSCALLS_OUTPUT_FILE` ->` cat "$SYSCALLS_OUTPUT_FILE"`

**Comment by Vincent43** [comment]
@curiosity-seeker  https://www.shellcheck.net/
```
$ shellcheck syscalls.sh

In syscalls.sh line 21:
strace -cfo $STRACE_OUTPUT_FILE "$@" && awk '{print $NF}' $STRACE_OUTPUT_FILE | sed '/syscall\|-\|total/d' | sort -u | awk -vORS=, '{ print $1 }' | sed 's/,$/\n/' > $SYSCALLS_OUTPUT_FILE
            ^-----------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                          ^-----------------^ SC2086: Double quote to prevent globbing and word splitting.
                                                                                                                                                                     ^-------------------^ SC2086: Double quote to prevent globbing and word splitting.


In syscalls.sh line 24:
cat $SYSCALLS_OUTPUT_FILE
    ^-------------------^ SC2086: Double quote to prevent globbing and word splitting.

For more information:
  https://www.shellcheck.net/wiki/SC2086 -- Double quote to prevent globbing ...

```

**Comment by Vincent43** [comment]
We could change `(pwd)` to `PWD` but nevermind. Merging as all feedback was addressed.

**Pull request merged**